expression/column.go

```diff
@@ -141,16 +141,13 @@ func (col *CorrelatedColumn) resolveIndices(_ *Schema) {
 
 // Column represents a column.
 type Column struct {
-	FromID      int
 	ColName     model.CIStr
 	DBName      model.CIStr
 	OrigTblName model.CIStr
 	TblName     model.CIStr
 	RetType     *types.FieldType
 	ID          int64
-	// Position means the position of this column that appears in the select fields.
-	// e.g. SELECT name as id , 1 - id as id , 1 + name as id, name as id from src having id = 1;
-	// There are four ids in the same schema, so you can't identify the column through the FromID and ColName.
+	// Position is the unique id of this column.
 	Position int
```

Comment:
I'll rename it to `UniqueID` or something after this pr.
I'll examine whether the `ID` of `Column` can be removed.